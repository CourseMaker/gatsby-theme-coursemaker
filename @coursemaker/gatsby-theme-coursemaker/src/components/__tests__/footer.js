import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '../footer';
import schoolDouble from '../__mocks__/school-double';

describe('Footer', () => {
    it('renders correctly', () => {
        // Created using the query from Header.js
        const schoolTestSample = schoolDouble;
        const tree = renderer.create(<Footer school={schoolTestSample} />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
