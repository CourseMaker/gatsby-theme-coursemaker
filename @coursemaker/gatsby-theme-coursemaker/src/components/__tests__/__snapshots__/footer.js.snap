// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<footer
  className="py-8 text-sm css-t471a7-Footer"
>
  <div
    className="container"
  >
    <div
      className="flex justify-between"
    >
      <p
        className="text-center text-gray-500"
      >
        Powered by
        <a
          className="font-bold css-dqoh9t-Footer"
          href="https://coursemaker.org"
        >
           
          CourseMaker
        </a>
      </p>
      <div
        className="flex ml-auto text-gray-500 space-x-5"
      >
        <a
          className="css-dqoh9t-Footer"
          href="/terms"
        >
          Terms
        </a>
        <a
          className="css-dqoh9t-Footer"
          href="/privacy"
        >
          Privacy
        </a>
      </div>
    </div>
  </div>
</footer>
`;
